stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.69.1
+version: 1.69.2
```

Comment:
We add a new feature, so I guess we should do a minor version bump?

Reply:
I've bumped to 1.70.0